test/hydrogen_all_electron.jl

```diff
@@ -1,19 +1,21 @@
 using Test
 using DFTK
+import DFTK: scf_convergence_density_difference
```

Comment:
just nitpicking, but in general I'm not a big fan of this import business, because it tends to get lost in refactorings. I'd rather do DFTK. when needed